apps/admission-webhook-k8s/kustomization.yaml

```diff
@@ -8,5 +8,7 @@ resources:
 - admission-webhook.yaml
 - binding.yaml
 - role.yaml
+- mutating-webhook-config.yaml
+- clusterspiffeid-webhook-template.yaml
```

Comment:
Is it correct to add Spire config into webhook deployment?
I guess our apps are hard-coupled with Spire anyway, so this should cause any real issues, but I feel like it would be cleaner to place it in the spire deployment.

Reply:
Yep, I agree, this is not the best solution, reworked